modules/sdk/src/main/scala/org/tessellation/sdk/domain/block/processing/BlockAcceptanceContext.scala

```diff
@@ -16,6 +16,8 @@ trait BlockAcceptanceContext[F[_]] {
 
   def getLastTxRef(address: Address): F[Option[TransactionReference]]
 
+  def getInitialTxRef: F[TransactionReference]
```

Comment:
Do you think we should keep the `F` context even though all implementations are actually pure and only wrap the value in `F`?

Reply:
it's not needed indeed. removed :+1: